.storybook/webpack.config.js

```diff
@@ -20,8 +22,14 @@ module.exports = ({ config }) => {
         require.resolve('babel-plugin-remove-graphql-queries'),
     ];
 
-    // Prefer Gatsby ES6 entrypoint (module) over commonjs (main) entrypoint
-    config.resolve.mainFields = ['browser', 'module', 'main'];
+    config.resolve = {
+        // Alias preview for Image component loading
+        alias: {
+            previewSetup: path.resolve(__dirname, '../preview/noop.js'),
```

Comment:
Added this in because the `Image` component (when making a story for BlockQuote) was trying to resolve this path, but it was undefined. The alias was just missing in this webpack config. I used the `noop` in previews here.